### PR TITLE
Add Ngrok example

### PR DESCRIPTION
**Pull request by Atlas16A**
Added Example Compose for using ngrok with server to bypass networks that block port forwarding.

### CODE REVIEW

File: examples/docker-compose-ngrok.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    volumes:
+      # attach the relative directory 'data' to the container's /data path
+      - ./data:/data
+
+  ngrok:
+    image: shkoliar/ngrok:latest
+    ports:
+      - 4551:4551
+    links:
+      - mc
```
Comment (itzg) [inline examples/docker-compose-ngrok.yml:22]: Did you happen to try without this? Docker compose network includes the name resolving of the other services by default, so `mc` should have already been resolvable.

**Review by itzg — APPROVED** [review]
Thanks! I'm good to merge this but was curious about my question first.

File: examples/docker-compose-ngrok.yml (hunk 1 above)
Comment (Atlas16A) [reply examples/docker-compose-ngrok.yml:22]: Id does appear to work without it, however I left it in as that's how the creator of the image suggested usage is setup, I assume its there for redundancy sake. Up to you though if ya wanna remove it.

Comment (itzg) [reply examples/docker-compose-ngrok.yml:22]: Yeah, it would be great to remove it.

**Review by itzg — APPROVED** [review]

**Pull request merged**